automated_packaging/prepare_release.pl

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/perl
+use lib '/usr/local/bin';
+use common_functions;
+use JSON;
+
+$NEW_VERSION = $ARGV[0];
+$PROJECT = $ARGV[1];
+$LAST_DATE = $ARGV[2];
+
+# Cherry-pick commit using commit end-points
+sub cherry_pick_commits {
+	@commit_urls = @_;
```

Comment:
In some places we used tabs, in some other we used spaces. Let's pick one and use it everywhere.